aip-console-jenkins/src/main/java/io/jenkins/plugins/aipconsole/AddVersionBuilder.java

```diff
@@ -493,12 +497,11 @@ protected void performClient(@Nonnull Run<?, ?> run, @Nonnull FilePath workspace
                 applicationService.updateModuleGenerationType(applicationGuid, requestBuilder, ModuleGenerationType.fromString(moduleGenerationType), !applicationHasVersion);
             }
 
-            boolean expandedSecurityDataflow = Boolean.valueOf(run.getEnvironment(listener).get("SECURITY_DATAFLOW"));
-
             requestBuilder.objectives(VersionObjective.BLUEPRINT, isBlueprint());
             requestBuilder.objectives(VersionObjective.SECURITY, expandedSecurityDataflow);
 
-            log.println("Update JEE and DOTNET security dataflow settings to: " + expandedSecurityDataflow);
+            //Settings.Option.Dataflow.info
```

Comment:
Since you said "ok" to the wondering I'd written regarding this property name in comment, I though it would have disappeared...

Reply:
hmm!